node-registrar/pkg/server/routes.go

```diff
@@ -21,39 +21,39 @@ func (s *Server) SetupRoutes() {
 	}))
 
 	s.router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
-	v1 := s.router.Group("v1")
```

Comment:
need to also support /v1 that we had because this will break the current nodes no?